src/pages/index.js

```diff
@@ -62,13 +134,53 @@ const cardSection = new Section(
   ".cards__list",
 );
 
-cardSection.renderItems();
-
 const editFormValidator = new FormValidator(config, profileEditForm);
 const addFormValidator = new FormValidator(config, addImageForm);
+const profileImageFormValidator = new FormValidator(config, profileImageForm);
+const profileEditFormValidator = new FormValidator(config, profileEditForm);
+
+profileImageFormValidator.enableValidation();
+
+function handleDeleteClick(cardToDelete) {
+  deleteConfirmationModal.open();
+
+  deleteConfirmationModal.setSubmitHandler(() => {
+    if (cardToDelete) {
+      api
+        .deleteCard(cardToDelete.id)
+        .then(() => {
+          cardToDelete.handleDeleteButton();
+          cardToDelete = null;
+          deleteConfirmationModal.close();
+        })
+        .catch((err) => console.error(err));
+    }
+  });
+}
+
+function handleLikeButton(card) {
+  if (card._isLiked) {
```

Comment:
`_isLiked` 

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.